war/src/main/webapp/help/system-config/master-slave/jnlpSecurity.html

```diff
@@ -1,45 +0,0 @@
-<div>
```

Comment:
Could you confirm this file is deleted because it's unused?

Reply:
See self-review.

Reply:
Confirmed in https://github.com/jenkinsci/jenkins/pull/3998#pullrequestreview-229096700